redis/src/cluster.rs

```diff
@@ -194,6 +187,13 @@ impl ClusterConnection {
         Ok(())
     }
 
+    /// Set an auto reconnect attribute.
```

Comment:
I don't see the point? This was already grouped with other `pub fn ..(&self)` methods, so not much point in moving it around within that group?

Reply:
I agree with you. There are no reason to move set_auto_reconnect. I restore it.

Reply:
Sure. I drop the commit.